var/ramble/repos/builtin/applications/hostname/application.py

```diff
@@ -18,9 +18,11 @@ class Hostname(ExecutableApplication):
 
     input_file('test', url="https://no.domain/file/test_dir.tgz", description="Example input file...")
 
-    executable('serial', '/usr/bin/time hostname', use_mpi=False)
-    executable('parallel', '/usr/bin/time hostname', use_mpi=True)
+    executable('local', 'time hostname', use_mpi=False, output_capture="&>")
```

Comment:
I'm not sure where we ended on this discussion, but can this use `OUTPUT.ALL` semantics instead of hard-coding `&>` ?

Reply:
updated